core/commands/keystore.go

```diff
@@ -202,6 +203,65 @@ var KeyListCmd = &cmds.Command{
 	Type: KeyOutputList{},
 }
 
+var keyRmCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Remove a keypair",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("name", true, true, "names of keys to remove"),
```

Comment:
Call on the return value of `StringArg` `.EnableStdin()` to enable stdin handling for it.